src/components/departureDetails/DepartureDetails.scss

```diff
@@ -0,0 +1,79 @@
+.DepartureDetails {
+    margin: 20px;
+    min-width: 300px;
+
+    .container {
+        display: flex;
+        flex-direction: column;
+        padding: 10px;
+
+        .departure, 
+        .destination {
+            display: flex;
+            & :first-child {
+                margin-right: 10px
+            }
+        }
+
+        .departure {
+            .time {
+                color: #207CCB;
+                font-weight: 600;
+                margin-right: 20px;
+            }
+            .city {
+                margin-right: 5px;
+                font-weight: 600
+            }
+        }
+
+        .destination {
+            color: #B1B3B5;
+            .time {
+                color: #B1B3B5;
+                font-weight: 600;
+                margin-right: 20px;
+            }
+            .city {
+                margin-right: 5px;
+                font-weight: 600
+            }
+        }
+
+        .bus {
+            margin: 5px;
+        }
+
+        .price {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin-top: 10px;
+
+            .passenger {
+                color: #F19433
+            }
+
+            .total {
+                color: #207CCB;
+                font-size: 24px;
+                align-self: flex-end;
+                margin-right: 15px;
+            }
+        }
+    } 
+}
+
+
+@media screen and (max-width:500px) {
+
+    .DepartureDetails {
+        .origin-location,
+        .destination-location{
+            white-space: nowrap;
+            width: 190px;
```

Comment:
oh i like how all numbers are multiplication of 5, would be better to use variables though